### ISSUE TITLE
Make `Term::prompt_multiline` support Alt+Backspace

### ISSUE THREAD

**Issue by avighnac** [jupyter-xeus/cpp-terminal#227]
On a normal Linux command line, Alt+Backspace erases an entire word. Supporting that would be a nice enhancement.

**Comment by MCWertGaming** [comment]
Yeah, we are currently working on improving the base functional and overall design. I sadly hadn't time yet to look into the prompt code. But I'll look into this when we are getting there, thanks!
